src/plays/popup-component/Readme.md

```diff
@@ -0,0 +1,32 @@
+# Popup component
+
+A simple popup/modal without libraries, packages, that can display texts, forms, buttons, Popup videos, any components in popup modal.
+
+### How it looks ( Example. a form)
+![Popup form](./images/popup.png)
+### A youtube embed
+![Youtube popup](./images/ytpopup.png)
+
+## Play Demographic
+
+- Language: js
+- Level: Intermediate
+
+## Creator Information
+
+- User: lifeofsoumya
+- Gihub Link: https://github.com/lifeofsoumya
+- Blog: https://turbohosty.com/blog/create-a-popup-in-wordpress-without-plugin/
+- Video: 
+
+## Implementation Details
+
+Update your implementation idea and details here
+
+## Consideration
+
+Update all considerations(if any)
+
+## Resources
```

Comment:
Here you can add what resources you used in this play like:

- React
- CSS

something like this.

It will be good when other users will read your play's `Readme`